lib/config-generator.js

```diff
@@ -403,27 +403,21 @@ class ConfigGenerator {
             // which can be useful for debugging, especially with HMR
             optimization.namedModules = true;
         }
+        // https://github.com/webpack/webpack/issues/8354
+        // likely can be removed in Webpack 5
+        // https://github.com/webpack/webpack/pull/8374
+        optimization.chunkIds = 'named';
 
         let splitChunks = {
             chunks: this.webpackConfig.shouldSplitEntryChunks ? 'all' : 'async'
         };
 
-        // hash the split filenames in production to protect exposing entry names
+        // This causes the split filenames (& internal names) to be,
+        // for example, 0.js. This is needed so that the filename
+        // doesn't change suddenly when another entry needs that same
+        // shared code (e.g. vendor~entry1~entry2.js).
```

Comment:
should this comment also include a link to the webpack issue where they suggested using `splitChunks.name = false;` ?